### PR TITLE
Fixed footer after changes in global styles.

### LINKED ISSUE #46: Update Footer Styles
With the changes to global styling, the footer no longer has the correct experience. Need to update.

### PR DESCRIPTION
**Pull request by sergiucoman**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #46

Test URLs:
- Before: https://main--troweprice-rf--hlxsites.hlx.page/
- After: https://bugfix-footer--troweprice-rf--hlxsites.hlx.page/

### CODE REVIEW

**Review by bstopp — CHANGES_REQUESTED** [review]

File: styles/styles.css [hunk 1]
```diff
@@ -79,7 +80,7 @@
 
 h1, h2, h3, h4, h5, h6 {
   margin-bottom: 1em;
-  font-weight: var(--font-weight-bold);
+  font-weight: var(--font-weight-normal);
```
[the remark this answers is not in the document]
Comment (sergiucoman) [reply styles/styles.css]: Right, I had mistakenly changed the global CSS. The change should have been done only in footer.css. Reverted the change.

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -112,29 +100,52 @@ footer .footer-main-content > div:first-child {
...
+    padding: 24px 1rem 3pc;
+    gap: 20px 2pc;
+  }
+
+  footer .footer-disclaimer-content {
+    padding: 0 1rem;
+  }
+}
+
+@media screen and (min-width: 960px) {
```
Comment (dnbute) [inline blocks/footer/footer.css:138]: Was this left over from before we switched to the 600/900/1200 break points or is there a specific reason for the 960px?

Comment (bstopp) [reply blocks/footer/footer.css:138]: Good Catch.

Comment (sergiucoman) [reply blocks/footer/footer.css]: yeah, good catch. I was using the breakpoints from the original site. It is fixed now.

**Review by dnbute — APPROVED** [review]
lgtm

**Review by bstopp — APPROVED** [review]

**Pull request merged**